Source/Core/Core/IOS/IPC.cpp

```diff
@@ -85,6 +85,282 @@ static u64 s_last_reply_time;
 
 static constexpr u64 ENQUEUE_REQUEST_FLAG = 0x100000000ULL;
 static constexpr u64 ENQUEUE_ACKNOWLEDGEMENT_FLAG = 0x200000000ULL;
+
+struct IosMemoryValues
+{
+  u16 ios_version;
+  u16 ios_revision;
+  u32 ios_date;
+  u32 mem1_physical_size;
+  u32 mem1_simulated_size;
+  u32 mem1_end;
+  u32 mem1_arena_begin;
+  u32 mem1_arena_end;
+  u32 mem2_physical_size;
+  u32 mem2_simulated_size;
+  u32 mem2_end;
+  u32 mem2_arena_begin;
+  u32 mem2_arena_end;
+  u32 ipc_buffer_begin;
+  u32 ipc_buffer_end;
+  u32 hollywood_revision;
+  u32 ram_vendor;
+  u32 unknown_begin;
+  u32 unknown_end;
+};
+
+static const u32 ADDR_MEM1_SIZE = 0x3100;
```

Comment:
These can all be constexpr. The `static` specifier can be dropped as well, if that's preferable, since const internally links by default.